### ISSUE TITLE
Community in "Resources"

### ISSUE THREAD

**Issue by JohnAnon9771** [sitepress/railsdocs#9]
I think one of the best ways to learn is with the help of the community, I think it would be beneficial to have a new card "community" where it would have links like:
- [Rails reddit](https://www.reddit.com/r/rails/)
- [Ruby reddit](https://www.reddit.com/r/ruby/)
- [Hotwired forum](https://discuss.hotwired.dev/)
- [Rails forum](https://discuss.rubyonrails.org/)

**Comment by bradgessler** [comment]
Agreed! If anybody wants to open a PR for this, copy what I did at https://github.com/sitepress/railsdocs/tree/main/app/content/pages/news, but put the pages in https://github.com/sitepress/railsdocs/tree/main/app/content/pages/communities.
